components/views/hrm/employees/employee-list/employeelist/Employeelist.jsx

```diff
@@ -10,11 +10,13 @@ import { ToggleSwitch } from 'crewhrm-materials/toggle-switch/ToggleSwitch.jsx';
 import { LoadingIcon } from 'crewhrm-materials/loading-icon/loading-icon.jsx';
 import { ContextToast } from 'crewhrm-materials/toast/toast.jsx';
 import { countries_object, employment_statuses, employment_types } from 'crewhrm-materials/data.jsx';
+import { DropDown } from 'crewhrm-materials/dropdown/dropdown.jsx';
 
 import EmployeelistCss from './employeelist.module.scss';
 import SearchImg from '../img/search-normal-add-8.svg';
 import empty_img from '../img/empty.png';
 
+export const employment_status_keys = Object.keys(employment_statuses);
```

Comment:
When a variable or function is not going to be used in another file, we should rather not export it.

Reply:
export removed.